app/components/agenda/agendaitem/agendaitem-controls/component.js

```diff
@@ -28,6 +33,14 @@ export default Component.extend(isAuthenticatedMixin, {
     });
   }),
 
+  async checkIfSubcaseExist(){
+      const subcase = await this.agendaitem.get('subcase');
+      if(!subcase) {
+        return;
+      }
+      return subcase;
```

Comment:
Same here. Your editor seems to go about 3 spaces for these async functions? That's strange!

Reply:
Can this be rewritten as the following?
```javascript
async checkIfSubcaseExist() {
  return await this.agendaitem.get('subcase');
},
```

Reply:
You don't have to, the contents just seemed like they don't really do much more than what can be reduced to a single line. If you like the more explicit version, we can keep it.

Reply:
This can be rewritten even better as:
```isPostPonable: computed("sessionService.agendas.@each","agendaitem.subcase", async function() {
    const subcase = await this.agendaitem.get('subcase');
    if(!subcase) {
      return false;
    }

  ....rest of the function
```

Reply:
Since there is no reuse of the
`checkIfSubcaseExist()`

Reply:
checkIfSubcaseExist() is named well for me. It could be reusable and it's perfectly allowed to create a function for this if it's only used once.